content/en/docs/collector/builder.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Builder
+weight: 25
+---
+
+The OpenTelemetry Collector Builder (ocb) is a command-line interface tool that
+generates sources and binaries for OpenTelemetry Collector
+[distributions](../distributions).
+
+This tool is currently being used to build both the core and the contrib
+distributions, and can be used to build custom distributions as well, allowing
+users to cherry-pick which components they need for their production usage,
+potentially even including their own custom components.
+
+## Installing
+
+You can download `ocb` from the OpenTelemetry Collector's
+[release page](https://github.com/open-telemetry/opentelemetry-collector/releases)
+on GitHub. A container image is **NOT** available at this time. Prefer to use
+the same version number that you'll use in your manifest, as the builder is only
+guaranteed to work with the OpenTelemetry Collector API matching the builder's
+version.
+
+## Manifest
+
+The central part of a custom distribution is the distribution's manifest. This
+manifest defines the metadata for the distribution and lists all components that
+should be included.
+
+This is an example `manifest.yaml`, containing only a couple of components:
+
+```yaml
+dist:
+  module: github.com/open-telemetry/opentelemetry-collector-releases/sidecar
+  name: sidecar
+  description: OpenTelemetry Collector Sidecar version
+  version: 0.73.0
```

Comment:
comments explaining how these values affect the output?

Reply:
Done!